packages/mui-codemod/src/deprecations/accordion-props/accordion-props.test.js

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from './accordion-props';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('deprecations', () => {
+    describe('accordion-props', () => {
+      it('transforms props as needed', () => {
+        const actual = transform(
+          {
+            source: read('./test-cases/actual.js'),
+            path: require.resolve('./test-cases/actual.js'),
```

Comment:
What's the difference between the `source` and the `path` properties? I couldn't find any reference to `path` inside the `transform` function.